acceptance/tests/ensure_version_file.rb

```diff
@@ -0,0 +1,23 @@
+require 'puppet/acceptance/temp_file_utils'
+extend Puppet::Acceptance::TempFileUtils
+
+# ensure a version file is created according to the puppet-agent path specification:
+# https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md
+
+test_name 'PA-466: Ensure version file is created on agent' do
+
+  skip_test 'requires version file which is created by AIO' if [:gem, :git].include?(@options[:type])
+
+  step "test for existence of version file" do
+    agents.each do |agent|
+      version_file = agent[:platform] =~ /windows/ ?
+        'C:/Program Files/Puppet Labs/Puppet/VERSION' :
```

Comment:
Yes, we need to look at `Program Files(x86)` when installing a 32-bit agent on 64-bit Windows.  Since Cygwin is running as 64-bit on a 64-bit OS, it can see both `Program Files` and `Program Files (x86)`.